ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -371,6 +374,23 @@ def add_capacity(osd_size_capacity_requested):
     storageDeviceSets->count = (capacity reqested / osd capacity ) + existing count storageDeviceSets
 
     """
+    lvpresent = check_local_volume()
+    if lvpresent:
+        path = os.path.join(constants.EXTERNAL_DIR, "device-by-id-ocp")
+        utils.clone_repo(constants.OCP_QE_DEVICEPATH_REPO, path)
+        os.chdir(path)
+        utils.run_cmd("ansible-playbook devices_by_id.yml")
+        dev_paths = utils.run_cmd(f"cat local-storage-block.yaml | grep {osd_size_capacity_requested}")
+        lvcr = get_local_volume_cr()
+        param = f"""[{{ "op": "replace", "path": "/spec/storageClassDevices/0/devicePaths",
+                        "value": {dev_paths}}}]"""
+        lvcr.patch(
+            resource_name=lvcr.get()['items'][0]['metadata']['name'],
+            params=param.strip('\n'),
+            format_type='json'
+        )
```

Comment:
please add a PV verification here make sure PV is created after patching local volume

Reply:
Added!